Mathlib/FieldTheory/Minpoly/IsIntegrallyClosed.lean

```diff
@@ -71,7 +71,7 @@ end
 
 variable [IsDomain S] [NoZeroSMulDivisors R S]
 
-variable [IsIntegrallyClosed R]
+variable [IsIntegrallyClosed R (FractionRing R)]
```

Comment:
Shouldn't we introduce a name for this? Can we call these rings "normal"?

Reply:
Normal implies `IsIntegrallyClosed R (FractionRing R)` but also IsReduced according to Stacks' definition.
Maybe we could call this generalized version `IsIntegrallyClosedIn` and let `IsIntegrallyClosed R := IsIntegrallyClosedIn R (FractionRing R)`, so the definition of `IsIntegrallyClosed` remains unchanged. To be honest I don't see much use of `IsIntegrallyClosedIn` yet ...